Config/Schema/schema.php

```diff
@@ -81,7 +81,7 @@ public function after($event = array()) {
 	public $cper_exchange_rates = array(
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "$cper_exchange_rates" to match the regular expression ^[a-z][a-zA-Z0-9]*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS116)